iceberg-catalog-migrator/cli/src/test/java/org/apache/polaris/iceberg/catalog/migrator/cli/AbstractCLIMigrationTest.java

```diff
@@ -106,11 +109,6 @@ private static void initializeCatalog(
     }
   }
 
-  @AfterAll
-  protected static void tearDown() throws Exception {
```

Comment:
Child class need to shutdown the container after drop table now due to use of docker. So, this function cannot be at parent class. Moved to child classes.